pipelines/powershell-package/templates/push.yaml

```diff
@@ -0,0 +1,19 @@
+# about this template: it contains steps needed for pushing the package
+
+# guidance for parameters:
+# - all of parameters should be passed from calling pipeline
+# - all parameters are replaced in the template when the template is loaded = it's a simple replace
+# - therefore all usages needs to be surrounded by ''
+
+parameters:
+  pushpackage: 'false'
+  nugetDevOpsFeedToPublish: '! need to be set by pipeline !' #sample value: '/123789456-aaaa-1111-1234-1237894560'
+
+steps:
+- ${{ if eq(parameters.pushpackage, 'true') }}:
+  - task: DotNetCoreCLI@2
+    displayName: 'NuGet push'
+    inputs:
+      command: push
+      packagesToPush: '$(Build.ArtifactStagingDirectory)/*.nupkg'
+      publishVstsFeed: '${{ parameters.nugetDevOpsFeedToPublish }}'
```

Comment:
is there a reason why not implement the step in a same way as /pipelines/nuget-package/templates/push.yaml = with a runtime "pushpackage" condition? or to change the step to be shared (like with a template parameter for packagesToPush)?